src/GitVersionExe/Program.cs

```diff
@@ -129,17 +129,27 @@ static void ConfigureLogging(Arguments arguments)
             {
                 try
                 {
-                    Directory.CreateDirectory(Path.GetDirectoryName(arguments.LogFilePath));
-                    if (File.Exists(arguments.LogFilePath))
+                    var logFileFullPath = Path.GetFullPath(arguments.LogFilePath);
+                    var logFile = new FileInfo(logFileFullPath);
+
+                    if (logFile.Directory == null)
```

Comment:
I don't mind not having the {}, but is the exception correct? That contradicts the code below.